frontend/packages/dev-console/src/components/topology2/TopologyPage.tsx

```diff
@@ -86,16 +87,31 @@ const TopologyPage: React.FC<Props> = ({
         variant={showListView ? NamespacedPageVariants.light : NamespacedPageVariants.default}
         hideApplications={showListView}
         toolbar={
-          <Tooltip position="left" content={showListView ? 'Topology View' : 'List View'}>
-            <Link
-              className="pf-c-button pf-m-plain"
-              to={`/topology/${namespace ? `ns/${namespace}` : 'all-namespaces'}${
-                showListView ? '' : '/list'
-              }`}
-            >
-              {showListView ? <TopologyIcon size="md" /> : <ListIcon size="md" />}
-            </Link>
-          </Tooltip>
+          <div style={{ display: 'flex' }}>
+            {!showListView && (
```

Comment:
It's a pity we can't have this behind "when there are workloads"... but that'd be too heavy. Maybe the future toolbar will solve this issue.

Reply:
I think it makes sense not to show this on the "all projects" view? cc @christianvogt

Reply:
@andrewballantyne Agreed, we should not show shortcuts on all projects view. Thanks for pointing this out.